packages/alert/src/Alert.tsx

```diff
@@ -0,0 +1,92 @@
+import CheckIcon from '@littlespoon/icons/lib/CheckIcon'
+import CloseIcon from '@littlespoon/icons/lib/CloseIcon'
+import ExclamationIcon from '@littlespoon/icons/lib/ExclamationIcon'
+import colors from '@littlespoon/theme/lib/colors'
+import type React from 'react'
+
+import {
+  AlertActionLink,
+  AlertCloseButton,
+  AlertDescription,
+  AlertMessages,
+  AlertTitle,
+  AlertWrapper,
+  VisuallyHidden,
+} from './AlertBase'
+
+export interface AlertProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The title of the component.
+   */
+  title?: string
+
+  /**
+   * The description of the component.
+   */
+  description?: string
+
+  /**
+   * The action link of the component.
+   */
+  actionLinkUrl?: string
+
+  /**
+   * The action link text of the component.
+   */
+  actionLinkText?: string
+
+  /**
+   * Callback when close button is clicked.
+   */
+  onClose?: () => void
+
+  /**
+   * The variant to use. Defaults to "success".
+   */
+  variant?: 'success' | 'warning' | 'critical' | 'informative'
```

Comment:
It could be helpful to have an enum defined for this since these strings are being compared in multiple places.

Reply:
Currently this is the only component that has these variants so I don't think it's necessary at the moment